server/src/main.rs

```diff
@@ -171,23 +191,151 @@ async fn main() -> Result<(), Error> {
     let backend = Backend::dynamo().await;
     #[cfg(debug_assertions)]
     let backend = if std::env::var_os("USE_DYNAMODB").is_some() {
-        Backend::dynamo().await
+        use aws_sdk_dynamodb::types::{PutRequest, WriteRequest};
```

Comment:
`main` got overloaded, we may want to move the seeding logic into the `Backend` impl block and, maybe, into a dedicated module, wdyt?

Reply:
I think I do actually want the seeding logic to live "out here", but let's at least pull it out into its own function that is fully under cfg. That way we can also more easily share the parsing of the JSON and such

Reply:
Refactored